index.mjs

```diff
@@ -1,8 +1,24 @@
-import * as models from './models';
-import { Importer, DirWatcher } from './libs';
+import http from 'http';
+import express from 'express';
 
-const watcher = DirWatcher.watch('./data', 5000);
+import registerMiddlewares from './middlewares';
+import appConfig from './config/app';
+import { setApi } from './routers';
 
-new Importer(models, watcher);
+const app = express();
```

Comment:
All the _express_ configuration should be moved to app.js file.
Afterall this file will contain the only 3 lines (+/- ones)
- Item **5** of the task